src/main/java/frc/robot/Constants.java

```diff
@@ -71,12 +76,19 @@
             public static final int kSpinnerMotorPort = 9; // motor for the wheel that spins the control panal
 
             public static final int kColorSensorPort = 0; // Color sensor that senses the colors on the control panel
+
+            //values the color sensor values to detect the colors on control panel
+            public static final int kRedTarget = 0; // color sensor red target value
+            public static final int kYellowTarget = 0; // color sensor yellow target value
+            public static final int kGreenTarget = 0; // color sensor green target value
+            public static final int kBlueTarget = 0; // color sensor blue target value
```

Comment:
The targets are actually Color objects.  Check out the [examples from REV](https://github.com/REVrobotics/Color-Sensor-v3-Examples/tree/master/Java) for how to declare them.